### ISSUE TITLE
Feature Request: aria-labelledby should have associated element in DOM

### ISSUE THREAD

**Issue by mohammedzamakhan** [mgechev/codelyzer#751]
https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_02

**Comment by mgechev** [comment]
Sounds good!

**Comment by wKoza** [comment]
we can also handle  aria-describedby which is important when we show an error message for instance.
